releasenotes/notes/prepare-2.2.3-1024be6b0494ce14.yaml

```diff
@@ -0,0 +1,4 @@
+---
+prelude: >
+    Qiskit 2.2.3 fixes a bug that was introduced by Qiskit 2.2.2 when
+    using a :class:`.PassManager` for more than one circuit.
```

Comment:
Minor nit: I would maybe say it's running a pass manager here rather than using. Or call out the run method as the ref instead